CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.16.1]
+### Fixed
```

Comment:
If we did want to include a revert explanation maybe something like this could work:

```suggestion
### Changed
- This release is a patch for v10.15.2, so all changes in v10.16.0 have been temporarily reverted. They will be restored in a future v16 release.

### Fixed
```